### PR TITLE
Add illustrations for CenterCrop and FiveCrop in gallery example

### PR DESCRIPTION
**Pull request by oke-aditya**
Helps #3688

Rendered [Transform docs](https://530931-73328905-gh.circle-artifacts.com/0/docs/transforms.html#torchvision.transforms.CenterCrop)

Rendered [Gallery docs](https://530931-73328905-gh.circle-artifacts.com/0/docs/auto_examples/plot_transforms.html#centercrop)

Hope it is fine.

cc @NicolasHug

### CODE REVIEW

File: gallery/plot_transforms.py [hunk 1]
```diff
@@ -46,6 +46,29 @@ def _plot(img, title, **kwargs):
 resized_img = T.Resize(size=30)(orig_img)
 plot(resized_img, "Resized image")
 
+####################################
+# CenterCrop
+# ----------
+# The :class:`~torchvision.transforms.CenterCrop` transform
+# (see also :func:`~torchvision.transforms.functional.center_crop`)
+# Crops the given image at the center.
```
Comment (NicolasHug) [inline gallery/plot_transforms.py:54]:
```suggestion
# crops the given image at the center.
```

File: gallery/plot_transforms.py [hunk 2]
```diff
@@ -46,6 +46,29 @@ def _plot(img, title, **kwargs):
...
+center_cropped_img = T.CenterCrop(size=(100, 100))(orig_img)
+plot(center_cropped_img, "Center Cropped Image")
+
+
+####################################
+# FiveCrop
+# --------
+# The :class:`~torchvision.transforms.FiveCrop` transform
+# (see also :func:`~torchvision.transforms.functional.five_crop`)
+# Crop the given image into four corners and the central crop.
```
Comment (NicolasHug) [inline gallery/plot_transforms.py:64]:
```suggestion
# crops the given image into four corners and the central crop.
```

File: gallery/plot_transforms.py [hunk 3]
```diff
@@ -46,6 +46,29 @@ def _plot(img, title, **kwargs):
...
 plot(resized_img, "Resized image")
 
+####################################
+# CenterCrop
+# ----------
+# The :class:`~torchvision.transforms.CenterCrop` transform
+# (see also :func:`~torchvision.transforms.functional.center_crop`)
+# Crops the given image at the center.
+center_cropped_img = T.CenterCrop(size=(100, 100))(orig_img)
+plot(center_cropped_img, "Center Cropped Image")
```
Comment (NicolasHug) [inline gallery/plot_transforms.py:56]: Nit: perhaps "Center crop" would be simpler? (same for the other 5 ones). No strong opinion though!

**Review by NicolasHug — APPROVED** [review]
Thanks @oke-aditya , some nits but LGTM!

File: gallery/plot_transforms.py (hunk 3 above)
Comment (oke-aditya) [reply gallery/plot_transforms.py:56]: Let's keep consistency since all use `_img` word I will add it every place.

Comment (NicolasHug) [reply gallery/plot_transforms.py:56]: I was mostly talking about the image title but I agree consistency is important!

**Comment by NicolasHug** [comment]
Noice! Thanks @oke-aditya

**Pull request merged**